dashing/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>cyclonedds</name>
+  <version>0.1.0</version>
+  <description>Performant and robust open-source DDS implementation.</description>
+  <maintainer email="logans@cottsay.net">Scott K Logan</maintainer>
```

Comment:
I don't believe it is a hard-and-fast rule and I can't remember the last time anyone actually ran scripts to check, but I think we may have a general preference for your Open Robotics email to be used here.

Reply:
That's the right call - I just forgot. Good catch!

Reply:
Updated.